lib/ts/recipe/multifactorauth/api/implementation.ts

```diff
@@ -0,0 +1,89 @@
+import Multitenancy from "../../multitenancy";
+import { APIInterface } from "../";
+import { MultiFactorAuthClaim } from "../multiFactorAuthClaim";
+import { getUser } from "../../..";
+import SessionError from "../../session/error";
+
+export default function getAPIInterface(): APIInterface {
+    return {
+        updateSessionAndFetchMfaInfoPUT: async ({ options, session, userContext }) => {
```

Comment:
change the name to resyncSessionAndFetchMFAInfoPUT

Reply:
Done